app/src/apdu_handler.c

```diff
@@ -38,6 +38,81 @@
 
 #include "view_internal.h"
 
+__Z_INLINE void handle_getversion(volatile uint32_t *tx) {
+#ifdef DEBUG
+    G_io_apdu_buffer[0] = 0xFF;
+#else
+    G_io_apdu_buffer[0] = 0;
+#endif
+    G_io_apdu_buffer[1] = LEDGER_MAJOR_VERSION;
+    G_io_apdu_buffer[2] = LEDGER_MINOR_VERSION;
+    G_io_apdu_buffer[3] = LEDGER_PATCH_VERSION;
+    G_io_apdu_buffer[4] = !IS_UX_ALLOWED;
```

Comment:
This is not needed anymore. This field is `device_locked`, therefore, we can hardcode it with false. The SDK won't reply any APDU message if the device is locked.